src/main/kotlin/app/revanced/patches/strava/patch/SubscriptionPatch.kt

```diff
@@ -0,0 +1,33 @@
+package app.revanced.patches.strava.patch
```

Comment:
Move this patch to `app.revanced.patches.strava.subscription.patch` as it is a subscription related patch.